# HG changeset patch
# User Rik <rik@octave.org>
# Date 1534197404 25200
#      Mon Aug 13 14:56:44 2018 -0700
# Branch stable
# Node ID e06ce02f4f74ae0284d20982a9deb92bced911a1
# Parent  2c37f2f827c58054e7d9f65cd62ec91f37381ce1
Don't print default value for "extent" property in manual (bug #54505).

* genpropdoc.m Don't print default for "extent" property for text and uicontrol objects.  Don't print default for "tightinset" property of axes.

diff --git a/doc/interpreter/genpropdoc.m b/doc/interpreter/genpropdoc.m
--- a/doc/interpreter/genpropdoc.m
+++ b/doc/interpreter/genpropdoc.m
@@ -800,16 +800,17 @@ box or \"out\".  __modemsg__.";
         s.doc = "Two-element vector @code{[2Dlen 3Dlen]} specifying the \
 length of the tickmarks relative to the longest visible axis.";
         s.valid = valid_2elvec;
 
       case "tightinset"
         s.doc = "Size of the @code{[left bottom right top]} margins \
 around the axes that enclose labels and title annotations.";
         s.valid = valid_4elvec;
+        s.printdefault = false;
 
       case "title"
         s.doc = "Graphics handle of the title text object.";
         s.valid = valid_handle;
 
       case "titlefontsizemultiplier"
         s.doc = "Ratio between the title fontsize and the tick \
 label fontsize";
@@ -1060,16 +1061,17 @@ z data.";
 
       case "editing"
         s.doc = doc_unused;
 
       case "extent"
         s.doc = "Vector @code{[x0 y0 width height]} indicating the size \
 and location of the text string.";
         s.valid = valid_4elvec;
+        s.printdefault = false;
 
       case "fontangle"
         s.doc = doc_fontangle;
 
       case "fontname"
         s.doc = doc_fontname;
         s.valid = valid_string;
 
@@ -1568,16 +1570,19 @@ point source (@qcode{\"local\"}).";
       ## Overridden shared properties
 
       ## Specific properties
       case "backgroundcolor"
       case "callback"
       case "cdata"
       case "enable"
       case "extent"
+        s.valid = valid_4elvec;
+        s.printdefault = false;
+
       case "fontangle"
         s.doc = doc_fontangle;
 
       case "fontname"
         s.doc = doc_fontname;
         s.valid = valid_string;
 
       case "fontsize"
